flowpm/utils.py

```diff
@@ -115,7 +115,7 @@ def cic_readout(mesh, part, name="CiCReadout"):
     value = tf.reduce_sum(weightedvals, axis=-1)
     return value
 
-def r2c3d(rfield, norm=None, dtype=tf.complex64, name="R2C3D"):
+def r2c3d(rfield, norm=None, dims=3, dtype=tf.complex64, name="R2C3D"):
```

Comment:
we probably should have a new function named `r2c2d` to handle the 2d case, just like tf.signal has both `fft3d` and `fft2d`

Reply:
Ok cool. I wasn't sure which way would be better, but that makes sense. I'll have that changed!